Courses/Views/Components/FeaturedItem.swift

```diff
@@ -1,59 +1,52 @@
 //
-//  HomeView.swift
+//  FeaturedItem.swift
 //  Courses
 //
-//  Created by Dmytro Churakov on 07.10.2022.
+//  Created by Dmytro Churakov on 11.10.2022.
 //
 
 import SwiftUI
 
-struct HomeView: View {
+struct FeaturedItem: View {
```

Comment:
Now you can just init view with `Course` model and use it everywhere

Reply:
@yura-hb , do not quite understand what needs to be done

Reply:
Looks like everything is correct, I was confused with `coursers[0]`, it is better, that you wouldn't set default value here

Reply:
@yura-hb , got you, fixed!